qutebrowser/utils/utils.py

```diff
@@ -864,6 +864,63 @@ def unused(_arg):
     pass
 
 
+def parse_numsettxt_into_list(txtset, nummax):
+    """Parse the given numbers set in text format, and return it as a list of
+    numbers.
+
+    Args:
+        txtset: Numbers set specification. E.g. '1,5,8-last,32,2', where 'last'
+                is a special keyword that denotes that maximum allowable
+                number.
+        nummax: The number that will be used to replace 'last'
+    """
+    txtset = txtset.lower().replace('last', '%d' % (nummax))
+    l = set()
```

Comment:
Call this `numbers` or so instead - `l` is a bad name because it's easily mixed up with `I` depending on what font people are using.